test/convert/duplex.js

```diff
@@ -149,11 +149,12 @@ test('.duplex() can pipe to errored stream with Stream.pipeline()', async t => {
 	outputStream.destroy(cause);
 
 	await assertPromiseError(t, pipeline(inputStream, stream, outputStream), cause);
+	await t.throwsAsync(finishedStream(stream));
```

Comment:
Hi @ehmicky hope you are doing well!

I just came from the linked PR and I am wondering if you could let me know why this fix the tests (I tried to access the failure log but seems they aren't available anymore) cheers!